module/VuFind/sql/mysql.sql

```diff
@@ -75,9 +75,11 @@ CREATE TABLE `ratings` (
 /*!40101 SET character_set_client = utf8mb4 */;
 CREATE TABLE `oai_resumption` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
+  `token` varchar(255) NOT NULL,
   `params` text,
   `expires` datetime NOT NULL DEFAULT '2000-01-01 00:00:00',
-  PRIMARY KEY (`id`)
+  PRIMARY KEY (`id`),
+  UNIQUE KEY (`token`),
```

Comment:
You need to remove the trailing comma here; it is preventing MySQL databases from being created successfully.

```suggestion
  UNIQUE KEY (`token`)
```